src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,35 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+
+  constructor(name) {
+    this.name = name;
+    this.health = 100;
+    Animal.alive.push(this);
+  }
```

Comment:
```suggestion
  health = 100;
  constructor(name) {
    this.name = name;
    Animal.alive.push(this);
  }
```
Это свойство можно вынести за конструктор, так как оно стандартное для всех животных и не зависит от параметров